var/spack/repos/builtin/packages/muparser/package.py

```diff
@@ -22,19 +22,25 @@ class Muparser(Package):
     # https://github.com/beltoforion/muparser/pull/46
     patch("auto_ptr.patch", when="@2.2.5")
 
-    depends_on("cmake@3.1.0:", when="@2.2.6:", type="build")
+    variant("samples", default=True, description="enable samples")
+    variant("openmp", default=True, description="enable OpenMP support")
+    variant(
+        "wide_char", default=False, description="enable wide character strings in place of ASCII"
+    )
+    variant("shared", default=True, description="enable shared libs")
```

Comment:
These variants are not used for the generic build. I recommend either

* Forcing use of CMake for `@2.2.6:` and restricting these variants to later versions
* or: adding `when="build_system=cmake"` to these variant definitions